joomla/joomla-cms
# [com_fields] Update fields.php

Problem: com_fields category fields change event causing problem in Persian fa-IR.
with this change, all calendars fields with localized data type (like Jalali) are converted to standard date type and then submit form.

Pull Request for Issue #16628 .

Summary of Changes

replace line 333 and change this code:

element.form.submit();

to

Joomla.submitform('" . $section . ".reload', element.form);

Testing Instructions

1- make sure you have more than two category for articles(com_content)
2- make sure you have at least a field for articles with context: com_content.article in com_fields
3- change back-end language to Persian(fa-IR).
4- go to Content > Articles. in Persian language the path is: محتوا > مطالب
5- edit an article and change category field value to another one.

Expected result

before this change:
after change category field value all calendar fields value change to empty.

after change code and replace line 333 with new one:
don`t show any error and fully reload page after ajax call and old calendar field values are show correctly.
